src/Mailer/Email.php

```diff
@@ -1925,6 +1929,25 @@ protected function _renderTemplates($content)
             $rendered[$type] = rtrim($rendered[$type], "\n");
         }
 
+        /* Embed images inline in html templates */
+        if (!empty($rendered['html'])) {
+            $rendered['html'] = str_replace(['file:', 'file://', 'cid://'], 'cid:', $rendered['html']);
```

Comment:
In terms of ease of use, functional html template code on the local computer is the best solution.

I'm using Thunderbird with an HTML signature of this kind and see logical and simple want to use the same signature as a template for CakePHP mails.

In terms of performance any template system causes a performance degradation.